Shaders/Common/public/BasicStructures.fxh

```diff
@@ -137,12 +137,25 @@ struct CameraAttribs
     matrix mProjInv;
     matrix mViewProjInv;
 #endif
+    uint iFrameIndex;
```

Comment:
This is currently only used to compute the blue noise texture.
We will have to pass this through a separate buffer for now.

Reply:
This field has been removed